tests/tests/fixtures/tables/simple_products.rs

```diff
@@ -49,7 +49,7 @@ BEGIN;
     	index_name => 'bm25_search',
         table_name => 'bm25_search',
     	schema_name => 'paradedb',
-        key_field => '%s',
+        key_field => 'id',
```

Comment:
Did you mean to change this?

Reply:
Yes, I think we should just fix the `id` if we don't need it to change. Less variables in tests the better, and its easier to copy this string directly now to debug.